src/main/fc/core.c

```diff
@@ -611,7 +625,7 @@ void tryArm(void)
         //beep to indicate arming
         if (featureIsEnabled(FEATURE_GPS)) {
             GPS_reset_home_position();
-
+            canUseGPSHeading = false; // block use of GPS Heading in position hold after each arm, until quad can set IMU to GPS COG
```

Comment:
What about other features that may need the GPS heading? We can't just give pos hold all authority over this.

Reply:
Only functions that want a valid GPS-only heading need to use it.
At present, only position hold uses it, and then only when there is no Mag and the user chose to bypass requiring a mag.  It's to prevent a flyaway only in that specific situation.
Other GPS functions just go ahead as normal.  For example, GPS Rescue will function like 'normal', it flies off until the IMU gets oriented when there is no Mag and the IMU is disoriented.